saiph/inverse_transform_test.py

```diff
@@ -117,24 +118,108 @@ def test_inverse_transform_deterministic() -> None:
 )
 def test_inverse_from_coord_mca(
     wbcd_quali_df: pd.DataFrame,
-    wbcd_supplemental_coord: pd.DataFrame,
-    wbcd_supplemental_df: pd.DataFrame,
+    wbcd_supplemental_coord_quali: pd.DataFrame,
 ) -> None:
     """Check that inverse supplemental coordinates using MCA yield correct results.
 
     We use `use_max_modalities=False` to keep the data logical.
     We compare indicators of the distributions for each column.
     """
     model = fit(wbcd_quali_df, nf="all")
+    # wbcd_supplemental_df_quali was obtained using
+    # inverse_transform with wbcd_supplemental_coord_quali and seed=123
```

Comment:
I think this belongs in the docstring of the fixture in `conftest.py`. Also, `wbcd_supplemental_df_quali` does not exist anymore, no ?

Reply:
No, instead I use the original wbcd data frame in my test. Simpler than  `wbcd_supplemental_df_quali`